src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1005,6 +1005,14 @@ public void selectMsqGenerateOptionsFor(String generateFor, int questionNumber)
                 generateFor);
     }
 
+    /**
+     * Selects the points distribution scheme for a const sum question.
+     *
+     * @param pointsOption Value of "Total" will select total number of points for all options points distribution scheme,
```

Comment:
Captalise `T` in `total number of points for all options`, and `N` in `number of points per option`. Also wrap them in `<i></i>`. Like this -

> Value of "Total" will select `<i>`Total number of points for all options`</i>` points distribution scheme, or "PerOption" will select `<i>`Number of points per option`</i>` points distribution scheme.